python/README.md

```diff
@@ -4,14 +4,45 @@ This folder contains sample applications written in Python Programming Language
 These applications have been tested with python 2.7 and 3.0 versions on Mac OS X.
 
 ## iot-rest-api-sample.py
-This sample app creates one event each of type - custom, network and error events.
+This sample app creates and sends one event each of type - custom, network and error events.
 
-Below variables need to be set before running the application.
-```python
-iot[appKey]  //AppKey generated in the controller when creating new application
-iot[collectorUrl] //URL to validate or send the events to collector
+Below is usage information for the sample app. Provide `appkey` as an input along with any options as needed.
+
+```sh
+USAGE: python iot-rest-api-sample.py appkey [options]
+options:
+-c, --collectorurl <url> IoT Collector URL to which the beacons should be sent to.
+-u, --url <url>          URL to trigger network request and capture network event.
+-x, --request <command>  Specify the request method to url. It is set to GET by default.
```

Comment:
`-X` see what  I mean about putting usage in the readme?

Reply:
Agree. I do acknowledge that duplicate information creates problems. But I do see the value of putting it in README where it gives a good idea of what the program does even before downloading the code and running it.

Any better ideas in dealing with this?

Reply:
not really other than keeping them in sync :(

Reply:
sadly, no.
I suppose you could generate it but that has other issues :(